bench-tps/src/main.rs

```diff
@@ -46,10 +46,13 @@ fn create_client(
     target_node: Option<Pubkey>,
 ) -> Arc<dyn BenchTpsClient + Send + Sync> {
     match external_client_type {
-        ExternalClientType::RpcClient => Arc::new(RpcClient::new_with_commitment(
-            json_rpc_url.to_string(),
-            CommitmentConfig::confirmed(),
-        )),
+        ExternalClientType::RpcClient => {
+            info!("Airdrop RPC: {:?}", json_rpc_url);
```

Comment:
This address is for more than airdrops with the RPC client. Either move the logic to somewhere [like this](https://github.com/solana-labs/solana/blob/6972afe2c4ed4202268dd71dc7d223f8348b4d01/bench-tps/src/bench.rs#L1042), or rewrite. Maybe `JSON RPC URL:`

Reply:
If you prefer the first option (ie. your primary intent is to clarify where airdrops are being requested from), might need to add a new method to BenchTpsClient, something like:
```
fn rpc_addr(&self) -> String {
    self.addr()
}
```
with an override for that default for ThinClient: `self.rpc_client().url()`

I do think this could be helpful, but don't feel super strongly; up to you.